### ISSUE TITLE
Add Kubectl Installation at minikube docs

### ISSUE THREAD

**Issue by kumarankit999** [kubernetes/minikube#17430]
If we set up the Kubernetes, we first install the Minikube and then install Kubectl. Then only we are able to set up the Kubernetes. But, if we see the https://minikube.sigs.k8s.io/docs/start/ , if we have to start the cluster. Firstly, we have to set up the kubectl but, the command for kubectl "sudo snap kubectl --classic" is missing.
![image](https://github.com/kubernetes/minikube/assets/111564362/cff8d1ed-da96-4e5a-a246-777ad43cd3a4)

I think we can add the kubectl installation commands before starting the cluster or between 2 & 3 step through kubectl or can provide this link https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/ there. So, that anyone can play with minikube cluster very smoothly.

**Comment by afbjorklund** [comment]
Did you try `minikube kubectl --` ?

https://minikube.sigs.k8s.io/docs/handbook/kubectl/

**Comment by kumarankit999** [comment]
I think we can add the kubectl installation command or link of that page in between 2 & 3 step because on the 3rd step the kubectl is caming suddenly and get a new term kubectl for him. which makes not understandable for beginners.

**Comment by afbjorklund** [comment]
Just saying that you can use minikube to install kubectl...

How to _use_ it after that, that's another story:

https://kubernetes.io/docs/reference/kubectl/

But installation (with version and os/arch), that is automated.

**Comment by kumarankit999** [comment]
Yes @afbjorklund, this is also a way

**Comment by kumarankit999** [comment]
Should We work on this @afbjorklund  ??

**Comment by afbjorklund** [comment]
I don't think we need a separate step, to install kubectl... There could be some links to the k8s documentation, though.


But we don't need any specials like `snap install` or `brew install`, but just use dl.k8s.io (see [downloadkubernetes.com](https://downloadkubernetes.com))

**Comment by afbjorklund** [comment]
See this issue, for why it was simplified and automated:

* https://github.com/kubernetes/minikube/issues/3645

Installing `minikube` was bad enough, for most users...

And it's good if they don't have to do it _again_, for `kubectl`.

**Comment by kumarankit999** [comment]
Ok, but can we add the kubectl installation any reference there?

**Comment by afbjorklund** [comment]
There is one page for each OS (unfortunately), but it could be something like:

----

If you already have kubectl installed (see [documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/)), you can now use it to access your shiny new cluster:

`kubectl get po -A`

Alternatively, minikube can download the appropriate version of kubectl and you should be able to use it like this:

`minikube kubectl -- get po -A`

----

Then the upstream documentation will have all the different installation options.

**Comment by kumarankit999** [comment]
Ok, Thanks for telling me this above. I have cleared all my doubts.
